chrome_extension/vendor/mooltipass/mcCombinations.js

```diff
@@ -1217,7 +1252,7 @@ mcCombinations.prototype.retrieveCredentialsCallback = function (credentials) {
 	// Selectors are ordered by priority, first ones are more important.
 	BUTTON_SELECTORS = [
 		'td.custom-button-center:visible',
-		'[type="submit"]:visible, a[href^="javascript:"]:visible',
+		'[type="button"]:visible, [type="submit"]:visible, a[href^="javascript:"]:visible',
```

Comment:
that looks like a major change. What does CI say?

Reply:
should we put the submit button before the button button?

Reply:
@limpkin it doesn't matter because we sort buttons after by the distance, so they have equal priority